ray-operator/Makefile

```diff
@@ -82,12 +85,12 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 docker-image: ## Build image only
-	docker build -t ${IMG} .
+	${ENGINE} build -t ${IMG} .
```

Comment:
I am not familiar with `podman`. Does `podman` support the same commands as Docker?

Reply:
Yep, it's effectively a drop in replacement for Docker. Some people go as far as to add `alias docker=podman` onto their systems. The major difference between the two is that Docker runs as root, whereas podman runs rootless by default. So far I haven't noticed any issues due to running as rootless.